src/main/java/me/danjono/inventoryrollback/commands/Commands.java

```diff
@@ -26,7 +22,7 @@
 import me.danjono.inventoryrollback.gui.menu.PlayerMenu;
 import me.danjono.inventoryrollback.inventory.SaveInventory;
 
-public class Commands extends ConfigData implements CommandExecutor, TabCompleter {
```

Comment:
Are u sure removing that is correct 😅

Reply:
yes, https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/command/TabExecutor.html

but I am not sure if this interface is there in older versions of Bukkit...

Reply:
Yes, this interface is implemented in 1.8, enough ^^